### DISCUSSION TITLE
Theos icon cannot be seen in Github dark mode

### DISCUSSION THREAD

**Discussion by raspiduino** [theos/theos#650] [Questions] [answered]
Icon in Readme.md when using Github dark mode:
![image](https://user-images.githubusercontent.com/68118236/169645824-cdfdc4de-2f1c-49d4-91df-91b5e0e286fe.png)

**Comment by kirb** [comment, marked as answer]
I did actually make it respond to light/dark mode using some CSS inside the SVG, but it only works in Chrome. It seems `@media (prefers-color-scheme: dark)` doesn’t work inside CSS inside an SVG inside an `<img>` in Safari or Firefox. (I should file a bug about it with both browsers…)

**Reply by raspiduino** [reply]
I use Chrome (version 101) :))

**Reply by raspiduino** [reply]
Oh it worked on my phone but not my PC (both using Chrome)

**Reply by PoomSmart** [reply]
Can we use `#gh-dark-mode-only` and `#gh-light-mode-only` in SVG URL?

**Reply by PoomSmart** [reply]
Or (recent update):

https://github.blog/changelog/2022-05-19-specify-theme-context-for-images-in-markdown-beta/

**Reply by kirb** [reply]
Oh wow, hot off the presses :grin: Thanks @PoomSmart! Changed it to that now.

@raspiduino, Chrome on iOS uses the WebKit engine (same one as Safari), rather than Blink (used on desktop and Android). That’s probably why it didn’t work for you on mobile.
